composer.json

```diff
@@ -42,8 +42,8 @@
 	},
 	"minimum-stability": "dev",
 	"scripts": {
-        "post-install-cmd": "./vendor/bin/cghooks add --ignore-lock",
-        "post-update-cmd": "./vendor/bin/cghooks update",
+        "post-install-cmd": "sh ./vendor/bin/cghooks add --ignore-lock",
+        "post-update-cmd": "sh ./vendor/bin/cghooks update",
```

Comment:
Windows does not have "sh", why is this necessary?

Reply:
If you use git bash there's a sh command and the hooks are added/updated after composer install/update.
Sadly ./ alone doesn't suffice.